datahub-web-react/src/app/entityV2/shared/tabs/Dataset/Schema/components/SchemaFieldDrawer/StatsTabWrapper.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import StatsSidebarView, {
+    StatsProps,
+} from '@app/entityV2/shared/tabs/Dataset/Schema/components/SchemaFieldDrawer/StatsSidebarView';
+import StatsTabContent from '@app/entityV2/shared/tabs/Dataset/Schema/components/SchemaFieldDrawer/StatsV2/StatsTabContent';
+import { useAppConfig } from '@src/app/useAppConfig';
+
+export default function StatsTabWrapper(props: StatsProps) {
+    const { config } = useAppConfig();
+    const { showStatsTabRedesign } = config.featureFlags;
+
+    return showStatsTabRedesign || true ? <StatsTabContent {...props} /> : <StatsSidebarView {...props} />;
```

Comment:
we need to remove this `|| true` hardcoded here same thing with in the other `StatsTabWrapper`